src/data/emacs-mode/agda2-mode.el

```diff
@@ -1378,15 +1378,17 @@ Either only one if point is a goal, or all of them."
                           'agda2-mimerAll))
 )
 
-(defun agda2-mimer ()
+(agda2-maybe-normalised-asis
+  agda2-mimer
   "Run proof search on a goal."
-  (interactive)
-  (agda2-goal-cmd "Cmd_autoOne" 'save 'goal))
+  "Cmd_autoOne"
+  nil
```

Comment:
Replacing `nil` by `'goal` seems to fix the regression that arguments are not passed to Mimer.